presto-cli/src/main/java/io/prestosql/cli/AbstractWarningsPrinter.java

```diff
@@ -13,7 +13,7 @@
  */
 package io.prestosql.cli;
 
-import io.prestosql.spi.PrestoWarning;
```

Comment:
Maybe change commit message to say something like "Replace PrestoWarning with new client Warning in JDBC driver"?